renovate.json

```diff
@@ -0,0 +1,55 @@
+{
+  "extends":[
+    "config:base"
+  ],
+  "ignorePaths": [
+    "**/node_modules/**",
```

Comment:
Those paths do not exist in this repo. Should we therefore omit them and re-add when required?

Reply:
Sorry I should have explained this actually but forgot about it. These are the defaults from `config:base` but with `**/test/**` removed since we do have code in that folder that I think we want to include. I'm open to reducing this list even more but didn't see any harm in keeping it.
Do you think we should remove it all? Or are there some folders we want to ignore?

Reply:
I'd agree to remove them all and add exclusions when we notice that they are helpful.